apps/service-portal/src/environments/environment.prod.ts

```diff
@@ -1,18 +1,22 @@
+declare global {
+  interface Window {
+    ENV: {
+      IDENTITY_SERVER_ISSUER_URL?: string
+      NX_DOCUMENT_PROVIDER_ADMINS?: string
+    }
+  }
+}
+
 export default {
   production: true,
   identityServer: {
-    IDENTITY_SERVER_ISSUER_URL:
-      window.location.origin === 'https://beta.staging01.devland.is'
-        ? 'https://identity-server.staging01.devland.is'
-        : window.location.origin === 'https://island.is'
-        ? 'https://innskra.island.is'
-        : 'https://identity-server.dev01.devland.is',
+    IDENTITY_SERVER_ISSUER_URL: window.ENV.IDENTITY_SERVER_ISSUER_URL,
   },
   sentry: {
     dsn:
       'https://3c45a55273774b91a897b85e0a1243d1@o406638.ingest.sentry.io/5501494',
   },
-  documentProviderAdmins: process.env.NX_DOCUMENT_PROVIDER_ADMINS || '',
+  documentProviderAdmins: window.ENV.NX_DOCUMENT_PROVIDER_ADMINS || '',
```

Comment:
This is a secret in helm, is it a secret? if so it should not be in the service-portal as it will be bundled to the client.